Essentials/src/com/earth2me/essentials/Mob.java

```diff
@@ -154,7 +159,8 @@ public Entity spawn(final World world, final Server server, final Location loc)
     public enum Enemies {
         FRIENDLY("friendly"),
         NEUTRAL("neutral"),
-        ENEMY("enemy");
+        ENEMY("enemy"),
+        ADULT_ENEMY("adult_enemy");
```

Comment:
Pretty sure this was added on accident, we shouldn't need it
```suggestion
        ;
```

Reply:
Was added for a reason, forgot to use it :P